unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/Tests/NFTShapeVisualTests.cs

```diff
@@ -120,7 +121,7 @@ IEnumerator InstantiateNFTShape(Vector3 position, int styleIndex)
 
         // Override texture with a local test one
         var nftShape = Environment.i.world.state.GetLoaderForEntity(entity) as LoadWrapper_NFT;
-        nftShape.loaderController.UpdateTexture(Resources.Load<Texture2D>("TestTexture"));
```

Comment:
Im leaving an a comment here but for another test. 

Would you mind adding an [Ensure ](https://github.com/decentraland/unity-renderer/blob/850c16873acf694fe28a2fabc6a0bb44ad63a8df/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Addressables/Tests/AddressablesEssentialsTests.cs#L16)Test for the Essentials assets? In that way, we know that those assets are going to be build and deployed

Those are currently EditorTests. But since we ll be using them soon, Im already migrating them